io_export_cryblend/export.py

```diff
@@ -745,20 +745,30 @@ def __process_bone_weights(self, object_, armature, skin_node):
         vw = ""
         vertex_groups_lengths = ""
         vertex_count = 0
+        vertex_group_count = 0

```

Comment:
Looks good, the only thing I see is that `vertex_group_count` doesn't need to be defined out here since it is already defined inside the for loops below.